collector/lib/GetKernelObject.cpp

```diff
@@ -40,8 +40,9 @@ extern "C" {
 
 namespace collector {
 
-const int kNumDownloadRetries = 90;
-const int kMaxDownloadRetriesTime = 90;
+const int kMaxDownloadRetriesTime = 180;
+const int kMaxDownloadRetriesInterval = 5;
+const int kNumDownloadRetries = kMaxDownloadRetriesTime / kMaxDownloadRetriesInterval;
```

Comment:
The assumption is that `max time = max retries * retry interval`, but isn't `max time = max retries * retry interval + processing time` is more correct? It doesn't break anything, but affects a bit many retries will be reached, right?

Reply:
yeah I suppose that's a fair assessment, though I'm not sure the processing time tends to be long enough to add too much variation in the overall wait time